randovania/games/fusion/exporter/patch_data_factory.py

```diff
@@ -193,27 +193,37 @@ def _create_nav_text(self) -> dict:
             self.patches.configuration, self.patches.game, self.patches
         )
         starting_items_text = (
-            "HQ has provided you with the following starting items: " + ", ".join(starting_items_list) + ". "
-            if len(starting_items_list) > 0
-            else ""
+            "Starting items: " + ", ".join(starting_items_list) + ". "
+            if self.configuration.short_intro_text
+            else "HQ has provided you with the following starting items: " + ", ".join(starting_items_list) + ". "
         )
+        if len(starting_items_list) == 0:
+            starting_items_text = ""
         metroid_location_text = "anywhere" if self.configuration.artifacts.prefer_anywhere else "at bosses"
+        long_intro = (
+            f"{starting_items_text}Your objective is as follows: the [COLOR=3]SA-X[/COLOR] "
+            f"has discovered and destroyed a top secret [COLOR=3]Metroid[/COLOR] breeding facility. "
+            f"It released {self.configuration.artifacts.placed_artifacts} "
+            "infant Metroids into the station. "
+            f"Initial scans indicate that they are hiding {metroid_location_text}. "
+            f"Find and capture {self.configuration.artifacts.required_artifacts} of them, "
+            "to lure out the SA-X. "
+            "Then initiate the station's self-destruct sequence. "
+            "Uplink at [COLOR=2]Navigation Rooms[/COLOR] along the way. "
+            "I can scan the station for useful equipment from there.[OBJECTIVE]Good. Move out."
+        )
+        short_intro = (
+            f"{starting_items_text}Gather {self.configuration.artifacts.required_artifacts}"
+            f"/{self.configuration.artifacts.placed_artifacts} Infant Metroids to lure out "
+            "the [COLOR=3]SA-X[/COLOR] and prepare for battle."
```

Comment:
AFAIK we have hint colors defined in the hint exporter. can you use those instead so that its clearer which colors these are supossed to be?